esp/src/eclwatch/TargetSelectWidget.js

```diff
@@ -170,17 +170,23 @@ define([
             }
         },
 
-        loadGroups: function () {
+        loadClusterGroups: function () {
             var context = this;
             WsTopology.TpGroupQuery({
                 load: function (response) {
                     if (lang.exists("TpGroupQueryResponse.TpGroups.TpGroup", response)) {
                         var targetData = response.TpGroupQueryResponse.TpGroups.TpGroup;
                         for (var i = 0; i < targetData.length; ++i) {
-                            context.options.push({
-                                label: targetData[i].Name,
-                                value: targetData[i].Name
-                            });
+                            switch(targetData[i].Kind) {
+                                case "Thor":
+                                case "hthor":
```

Comment:
No good reason afaics.. just legacy constant strings used in the creation of Group(s) during Dali startup from the cluster topologies in the Environment.